backend/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_ADMIN_TOKEN=simple_example_token
```

Comment:
This token will only be used on the backend, so it should be `ADMIN_TOKEN` without `REACT_APP`. If we add an admin interface, we'll have to add a separate authentication mechanism anyways.

Reply:
The env variable name must have the REACT_APP prefix, because it won't otherwise be imported 🤡 https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
Yes, but the token is not used by the frontend and it shouldn’t be either, as that would spill the secret token to the public 🤡

Reply:
That's why the `.env` file is in the `backend/` directory? And is simple `simple_example_token` so that the user would change it to something else?

Reply:
As the token is never used in the frontend there is no need to have the `REACT_APP` prefix. I would just use the `ADMIN_TOKEN` for now and then rethink it once/if we add some type of admin interface to the frontend

Reply:
Apparently the dotenv included the `ADMIN_TOKEN` in the `.env` file without the `REACT_APP_` prefix, so I pushed now a commit with the requested change.

Reply:
Also, ADMIN_TOKEN given via command line overwrites the defined ADMIN_TOKEN in the `.env` file 👌